node/src/indexer/fake.rs

```diff
@@ -215,6 +216,46 @@ impl FakeMpcContractState {
             }
         }
     }
+
+    pub fn change_participant_info(
```

Comment:
Nit: Not sure why I react to this. Just pure intuition/habit. Feel free to disagree/ignore.

```suggestion
    pub fn update_participant_info(
```